demos/python_demos/llm_text_generation/servable_unary/model.py

```diff
@@ -170,8 +170,11 @@ def execute(self, inputs: list):
             generate_kwargs["stopping_criteria"] = StoppingCriteriaList(stop_tokens)
 
         ov_model_exec = self.ov_model.clone()
+        token_count: List[int]= []
         def generate():
-            ov_model_exec.generate(**tokens, **generate_kwargs)
+            result = ov_model_exec.generate(**tokens, **generate_kwargs)
+            token_count.append(len([1 for x in result.numpy().flatten() if x not in tokenizer.convert_tokens_to_ids(tokenizer.all_special_tokens)]))
```

Comment:
token count includes tokens sent to the server in prompt. We should remove from the result the number of received tokens to track only generated tokens.